modules/client/shared/src/test/scala/com/azavea/stac4s/api/client/SttpEitherInstances.scala

```diff
@@ -0,0 +1,33 @@
+package com.azavea.stac4s.api.client
+
+import cats.effect.{ExitCase, Sync}
+
+trait SttpEitherInstances {
+
+  /** [[Sync]] instance defined for Either[Throwable, *].
+    * It is required (sadly) to derive [[fs2.Stream.Compiler]] which is necessary for the [[fs2.Stream.compile]] function.
+    */
+  implicit val eitherSync: Sync[Either[Throwable, *]] = new Sync[Either[Throwable, *]] {
```

Comment:
We use the default synchronous Either backend to use the same tests set for the Scala JS backend. (c)